### DISCUSSION TITLE
Copying only specific parts from one pattern to another

### DISCUSSION THREAD

**Discussion by djokenzi** [bangcorrupt/hacktribe#339] [Q&A] [answered]
Hi guys, 

This might not be a Hacktribe specific function (I'm really hoping there's a way to do it with original firmware as well), but I am struggling to find a way to achieve what I'm after.

I would like to copy only specific parts from one pattern to another. For example, I want to copy Part 7 from Pattern 001 to Part 7 on Pattern 099 (so, part only, not the whole pattern). 

Is there any way to do this? It doesn't matter if it involves connecting the device to a computer and using some kind of software to do it (if such program exists). Any solution would do, because I want to organize my patterns for a live set I'm preparing and would really like to avoid recreating each part.

Hope you can help!

**Comment by bangcorrupt** [comment, marked as answer]
This is a feature in the factory firmware.  Press [Shift]+[Pad 13], or go to Menu > Part Utility > Copy Part.

**Comment by djokenzi** [comment]
Hi bangcorrupt, 

Thanks for answering.

I'm not near my Electribe atm, but if memory serves me well that feature can only be used within a pattern. Does it really allow for parts to be copied from one pattern to another in such way?

I'll give it a go asap.

**Reply by richardarnezjames** [reply]
There is an [api](https://apiel.github.io/electribe-ui/dist/index.html) you can use for basic organization


There's also an old [max4live device](https://maxforlive.com/library/device/2816/electribe2-pattern-editor) that provides a lot deeper parameters

**Reply by bangcorrupt** [reply]
Yes, go to the destination pattern, then select the source pattern when running Copy Part.

**Reply by djokenzi** [reply]
Wow, thank you both! This will help a lot!
